config.py

```diff
@@ -2906,9 +2906,12 @@ def import_er_data(
     _drop_channels_func(cfg, raw=raw_er, subject='emptyroom', session=session)
 
     # Set same set of bads as in the experimental run, but only for MEG
-    # channels (because we won't have any others in empty-room recordings)
+    # channels (we might not have non-MEG channels in empty-room recordings).
     raw_er.info['bads'] = [ch for ch in bads if ch.startswith('MEG')]
 
+    # Only keep MEG channels.
+    raw_er.pick_types(meg=True)
```

Comment:
for CTF I assume you also need ref_meg=True

to be tested though

Reply:
The default for `pick_types` should be `'auto'` which means

> If ‘auto’, reference channels are included if compensations are present and meg is not False. 

So for CTF I think this will end up acting like True anyway